testing/skopeo/APKBUILD

```diff
@@ -0,0 +1,39 @@
+# Contributor: Ed Robinson <ed@reevoo.com>
+# Maintainer: Ed Robinson <ed@reevoo.com>
+pkgname=skopeo
+pkgver=0.1.23
+pkgrel=0
+pkgdesc="Work with remote images registries - retrieving information, images, signing content"
+url="https://github.com/projectatomic/skopeo"
+arch="x86_64"
```

Comment:
It should do, but I have no way of testing, I only have an Intel machine to test on.